stable/grafana/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: grafana
-version: 2.3.3
+version: 2.3.5
```

Comment:
as this is a breaking change, lets bump the major version.

Reply:
Do you mine 3.0.0?

Reply:
3.0.0 is correct!